deploy/yamls/lvmvolume-crd.yaml

```diff
@@ -108,6 +114,8 @@ spec:
             required:
             - capacity
             - ownerNodeID
+            - shared
+            - thinProvision
```

Comment:
we shouldn't be changing the schema, this may be a breaking change?

Reply:
Not sure what caused this. I updated the dependencies and regenerated the CRD files. `thinProvision` was added with https://github.com/openebs/lvm-localpv/pull/30 from 2021. Any suggestions on what to do?